semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/http/HttpSemanticConventionUtils.java

```diff
@@ -196,6 +197,8 @@ public static boolean isAbsoluteUrl(String urlStr) {
               attributeValueMap.get(HTTP_HOST.getValue()).getValue(),
               attributeValueMap.get(HTTP_TARGET.getValue()).getValue());
       return Optional.of(url);
+    } else if (attributeValueMap.containsKey(HTTP_URL.getValue())) {
```

Comment:
This is the fallback case, so should have the least priority?

Reply:
ok